plugin.video.hardstyletv/default.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#------------------------------------------------------------
+# License: GPL (http://www.gnu.org/licenses/gpl-3.0.html)
+# Based on code from youtube addon
+#------------------------------------------------------------
+
+import os
+import sys
+import plugintools
+import xbmc,xbmcaddon
+from addon.common.addon import Addon
+
+addonID = 'plugin.video.hardstyletv'
+addon = Addon(addonID, sys.argv)
+local = xbmcaddon.Addon(id=addonID)
```

Comment:
please use only `xbmcaddon.Addon()`. No need for the id (only if you were accessing other plugin from yours).